libs/platform/src/lib/components/form/form-group/form-field-group/form-field-group.component.ts

```diff
@@ -0,0 +1,43 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ContentChildren,
+    Input,
+    OnInit,
+    QueryList,
+    TemplateRef,
+    ViewChild,
+    ViewEncapsulation
+} from '@angular/core';
+import { FormField } from '../../form-field';
+import { FormGroupContainer} from '../../form-group';
+
+@Component({
+    selector: 'fdp-form-field-group',
+    template: `<ng-content></ng-content>`,
+    styleUrls: ['./form-field-group.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    encapsulation: ViewEncapsulation.None
+})
+export class FormFieldGroupComponent implements OnInit {
+    @Input()
+    label: string;
+
+    @ViewChild('renderer', { read: TemplateRef })
+    renderer: TemplateRef<any>;
+
+    @ContentChildren(FormField) groupFields: QueryList<any>;
```

Comment:
Could you document it - what's the reason of keeping it, just to anyone who sees this variable for a first time, to clarify a bit.

Also let's change `any` type